.circleci/perform-release-packages.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+git config credential.helper "store --file=.git/credentials"
+echo "https://$RELEASE_GH_TOKEN:@github.com/$TRAVIS_REPO_SLUG.git" > .git/credentials
+
+git config --global user.email "npmjs@commercetools.com"
+git config --global user.name "CT Release Bot"
+git config --global push.default simple
+
+npm config set //registry.npmjs.org/:_authToken=${NPM_TOKEN} -q
+
+echo "Running release:semantic"
+yarn release:semantic
```

Comment:
I would honestly try to get rid of this and simply use `lerna publish`. To be discussed...

Reply:
And I don’t mean as an automatic step but as a manual step

Reply:
See also #223, which didn’t really fix the problem. We have the tool there, we should just use it

Reply:
Lets have a ScreenHero chat about this. How to proceed etc, I can then chat with @wizzy25 or he joins in.